src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -70,6 +72,10 @@ class LocationResource extends Resource {
                   @QueryParam('giRestroom') Boolean giRestroom) {
 
         try {
+            if (pageSize > MAX_PAGE_SIZE) {
```

Comment:
Is this something that we should have as part of the skeleton? The various APIs could then overwrite the variable or use a configuration value to provide another value.